Libraries/src/Amazon.Lambda.PowerShellHost/PowerShellFunctionHost.cs

```diff
@@ -295,6 +296,7 @@ private void SetupStreamHandlers()
             this._ps.Streams.Information.DataAdding += _loggerFactory("Information");
             this._ps.Streams.Warning.DataAdding += _loggerFactory("Warning");
             this._ps.Streams.Error.DataAdding += _loggerFactory("Error");
+            this._ps.Streams.Debug.DataAdding += _loggerFactory("Debug");
```

Comment:
nit: These lines are in order or log level. Can you put `Debug` in between `Verbose` and `Information`? Also the block above where you are clearing can we put those clears in the same order. I think it is easier for readers.

Reply:
Done